<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="top_left_cross_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="top_left_zero_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="middle_left_cross_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="middle_left_zero_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="down_left_cross_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="down_left_zero_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="top_middle_cross_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="top_middle_zero_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="middle_middle_cross_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="middle_middle_zero_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="down_middle_cross_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="down_middle_zero_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="top_right_cross_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_right_zero_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="middle_right_cross_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="middle_right_zero_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="down_right_cross_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/крестик.webp</normaloff>../Users/sanya/Downloads/крестик.webp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="down_right_zero_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Users/sanya/Downloads/нолик.jpg</normaloff>../Users/sanya/Downloads/нолик.jpg</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>107</width>
     <height>107</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
